sql-files/item_db_etc.sql

```diff
@@ -515,7 +515,7 @@ REPLACE INTO `item_db` (`id`,`name_aegis`,`name_english`,`type`,`price_buy`,`wei
 REPLACE INTO `item_db` (`id`,`name_aegis`,`name_english`,`type`,`price_buy`,`weight`,`location_right_accessory`,`location_left_accessory`,`flag_buyingstore`,`script`) VALUES (4228,'Sleeper_Card','Sleeper Card','Card',20,10,true,true,true,'bonus3 bAddMonsterDropItem,12031,RC_Fish,100;');
 REPLACE INTO `item_db` (`id`,`name_aegis`,`name_english`,`type`,`price_buy`,`weight`,`location_head_top`,`location_head_mid`,`location_head_low`,`flag_buyingstore`,`script`) VALUES (4229,'C_Tower_Manager_Card','Tower Keeper Card','Card',20,10,true,true,true,true,'bonus bInt,1; bonus bCastrate,-5;');
 REPLACE INTO `item_db` (`id`,`name_aegis`,`name_english`,`type`,`price_buy`,`weight`,`location_right_accessory`,`location_left_accessory`,`flag_buyingstore`,`script`) VALUES (4230,'Shinobi_Card','Shinobi Card','Card',20,10,true,true,true,'bonus bAgi,1; bonus3 bAutoSpellWhenHit,"AS_CLOAKING",5,100;');
-REPLACE INTO `item_db` (`id`,`name_aegis`,`name_english`,`type`,`price_buy`,`weight`,`location_left_hand`,`flag_buyingstore`,`script`) VALUES (4231,'Increase_Soil_Card','Mi Gao Card','Card',20,10,true,true,'bonus2 bSubRace2,RC2_Guardian,50;');
+REPLACE INTO `item_db` (`id`,`name_aegis`,`name_english`,`type`,`price_buy`,`weight`,`location_left_hand`,`flag_buyingstore`,`script`) VALUES (4231,'Increase_Soil_Card','Mi Gao Card','Card',20,10,true,true,'bonus2 bAddDamageClass,1285,-50; bonus2 bAddDamageClass,1286,-50; bonus2 bAddDamageClass,1287,-50; bonus2 bAddDamageClass,1899,-50; bonus2 bAddDamageClass,1900,-50;');
 REPLACE INTO `item_db` (`id`,`name_aegis`,`name_english`,`type`,`price_buy`,`weight`,`location_right_accessory`,`location_left_accessory`,`flag_buyingstore`,`script`) VALUES (4232,'Wild_Ginseng_Card','Hermit Plant Card','Card',20,10,true,true,true,'bonus2 bAddItemGroupHealRate,IG_Herb,50; bonus3 bAddMonsterDropItem,507,RC_Plant,300; bonus3 bAddMonsterDropItem,508,RC_Plant,200; bonus3 bAddMonsterDropItem,509,RC_Plant,100;');
```

Comment:
guys, code is probably good but why does a refactor job have some value edited ?
Couldn't we have a strickly mobdb.txt to moddb.yaml and then a new PR with those update db value ?
I mean it's a itemdb update.. what about if someone wanted to cherry-pick those itemdb but not the mobdbyaml or visversa?

Reply:
The edited item value is because of the adjustment of the RC2 setup. While working on this I noticed that this item wasn't using the official format, which Aegis just defines the specific mob ID.